AUDIO/PlayPain.md

```diff
@@ -5,65 +5,55 @@ ns: AUDIO
-`PLAY_PAIN(Ped ped, int painID, float p3, bool createNetEvent)`
-
-Check the *examples* section for the correct usage of this native.
-
-**Description:**
-
-Plays a pain sound. A maximum of 33 pain IDs are allowed.  
-`createNetEvent` creates a `CPedPlayPainEvent` when set to true.
-
-Below is a list of all the pain IDs (Asterisks indicate that sounds can play in a Low, Medium or High fashion), for example: `PAIN_LOW_GENERIC`. The corresponding pain strings belong to the game exe.
-
-**Pain IDs:**
-
-- 0: PAIN_*_GENERIC (Low, Medium, High)
-- 1: UNUSED
-- 2: UNUSED
-- 3: SCREAM_PANIC (Nothing can be heard)
-- 4: SCREAM_PANIC_SHORT
-- 5: SCREAM_SCARED
-- 6: SCREAM_SHOCKED
-- 7: SCREAM_TERROR
-- 8: ON_FIRE
-- 9: UNUSED
-- 10: UNUSED
-- 11: INHALE (Nothing can be heard)
-- 12: EXHALE (Nothing can be heard)
-- 13: DEATH_HIGH_SHORT
-- 14: UNUSED
-- 15: PAIN_HIGH_GENERIC
-- 16: PAIN_*_GENERIC (Low, Medium, High)
-- 17: PAIN_SHOVE
-- 18: PAIN_WHEEZE
-- 19: COUGH
-- 20: PAIN_TAZER
-- 21: UNUSED
-- 22: CLIMB_LARGE (Nothing can be heard)
-- 23: CLIMB_SMALL (Nothing can be heard)
-- 24: JUMP (Nothing can be heard)
-- 25: COWER
-- 26: WHIMPER
-- 27: DYING_MOAN
-- 28: EXHALE_CYCLING (Nothing can be heard)
-- 29: PAIN_RAPIDS (Nothing can be heard)
-- 30: SNEEZE
-- 31: MELEE_SMALL_GRUNT (Nothing can be heard)
-- 32: MELEE_LARGE_GRUNT (Nothing can be heard)
-- 33: PAIN_*_GENERIC (Low, Medium, High) 
+```c
+enum eAudDamageReason {
+	AUD_DAMAGE_REASON_DEFAULT = 0,
+	AUD_DAMAGE_REASON_FALLING = 1,
+	AUD_DAMAGE_REASON_SUPER_FALLING = 2,
+	AUD_DAMAGE_REASON_SCREAM_PANIC = 3,
+	AUD_DAMAGE_REASON_SCREAM_PANIC_SHORT = 4,
+	AUD_DAMAGE_REASON_SCREAM_SCARED = 5,
+	AUD_DAMAGE_REASON_SCREAM_SHOCKED = 6,
+	AUD_DAMAGE_REASON_SCREAM_TERROR = 7,
+	AUD_DAMAGE_REASON_ON_FIRE = 8,
+	AUD_DAMAGE_REASON_DROWNING = 9,
+	AUD_DAMAGE_REASON_SURFACE_DROWNING = 10,	// drowning on the surface of water, after we time out
+	AUD_DAMAGE_REASON_INHALE = 11,
+	AUD_DAMAGE_REASON_EXHALE = 12,
+	AUD_DAMAGE_REASON_POST_FALL_GRUNT = 13,
+	AUD_DAMAGE_REASON_ENTERING_RAGDOLL_DEATH = 14,
+	AUD_DAMAGE_REASON_EXPLOSION = 15,
+	AUD_DAMAGE_REASON_MELEE = 16,
+	AUD_DAMAGE_REASON_SHOVE = 17,
+	AUD_DAMAGE_REASON_WHEEZE = 18,
+	AUD_DAMAGE_REASON_COUGH = 19,
+	AUD_DAMAGE_REASON_TAZER = 20,
+	AUD_DAMAGE_REASON_EXHAUSTION = 21,
+	AUD_DAMAGE_REASON_CLIMB_LARGE = 22,
+	AUD_DAMAGE_REASON_CLIMB_SMALL = 23,
+	AUD_DAMAGE_REASON_JUMP = 24,
+	AUD_DAMAGE_REASON_COWER = 25,
+	AUD_DAMAGE_REASON_WHIMPER = 26,
+	AUD_DAMAGE_REASON_DYING_MOAN = 27,
+	AUD_DAMAGE_REASON_CYCLING_EXHALE = 28,
+	AUD_DAMAGE_REASON_PAIN_RAPIDS = 29,
+	AUD_DAMAGE_REASON_SNEEZE = 30,
+	AUD_DAMAGE_REASON_MELEE_SMALL_GRUNT = 31,
+	AUD_DAMAGE_REASON_MELEE_LARGE_GRUNT = 32,
+	AUD_DAMAGE_REASON_POST_FALL_GRUNT_LOW = 33
+}
+```
 
 ## Parameters
 * **ped**: The ped entity.
-* **painID**: The pain id, max allowed id is 33.
+* **damageReason**: The damage reason/type, refer to eAudDamageReason
+* **rawDamage**: Damage value, defaults to 0.0
```

Comment:
I documented this in 2022, cannot believe it had a mismatching parameter count all this long (scratching head).